.eslintrc.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
     visit: true,
     waitForLoadedIFrame: true,
     waitForUnloadedIFrame: true,
-    '$': true
+    '$': true,
+    CodeMirror: false
```

Comment:
Why is this false?

Reply:
To access [here](https://github.com/ember-cli/ember-twiddle/pull/622/files#diff-2de9eda1655d499e934e7465e87ca0bdR8)